src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,33 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    static final int MIN_AGE = 18;
+    static final int MIN_PASSWORD_LENGTH = 6;
 
     @Override
     public User register(User user) {
-        return null;
+        StorageDao storageDao = new StorageDaoImpl();
```

Comment:
storageDao is a dependency of the RegistrationService class and the register method shouldn't be responsible for creating it